primitives/Cargo.toml

```diff
@@ -16,6 +16,11 @@ sp-core = { default-features = false, git = "https://github.com/Cardinal-Cryptog
 sp-runtime = { default-features = false, git = "https://github.com/Cardinal-Cryptography/substrate.git", branch = "liminal-v0.9.32" }
 sp-std = { default-features = false, git = "https://github.com/Cardinal-Cryptography/substrate.git", branch = "liminal-v0.9.32" }
 sp-staking = { default-features = false, git = "https://github.com/Cardinal-Cryptography/substrate.git", branch = "liminal-v0.9.32" }
+sp-runtime-interface = { default-features = false, git = "https://github.com/Cardinal-Cryptography/substrate.git", branch = "liminal-v0.9.32" }
+
+ark-poseidon = { default-features = false, path="../poseidon", package = "liminal-ark-poseidon" }
```

Comment:
I'm slightly against this naming, as it strongly suggests that this crate is from arkworks :| if you don't like the long version, maybe just `poseidon`?

Reply:
I cannot call it poseidon since runtime interface creates module with the same name.

Reply:
`liminal-ark-poseidon`? :P

Reply:
I can feel that you tried to get your PR merged recently and somebody picked on you hard...
done

Reply:
xD